src/EnergyPlus/CondenserLoopTowers.hh

```diff
@@ -71,7 +71,7 @@ namespace CondenserLoopTowers {
     enum class ModelType
     {
         // Empirical Model Type
-        Unassigned,
+        Unassigned = -1,
         CoolToolsXFModel,
```

Comment:
Given that `enum class` is a name scope, avoid including the name of the type in the name of any of its values, e.g., `Model` here. Otherwise, the resolved name is `ModelType::CoolToolsXFModel`.